### ISSUE TITLE
Cmake needs position independant code switched on

### ISSUE THREAD

**Issue by memecode** [sammycage/lunasvg#200]
When I try and build using cmake and then use in my projects. It complains that -fPIC isn't turned on. This can be fixed by adding:
`set(CMAKE_POSITION_INDEPENDENT_CODE ON)`

To the cmake file, up the top somewhere.

**Comment by sammycage** [comment]
Yeah, that's the fix! The whole idea is to enable it whenever the compiler complains about needing `-fPIC`. If you're building from the command line, you can also just pass `-D CMAKE_POSITION_INDEPENDENT_CODE=ON` to cmake. That should sort it out!

**Comment by memecode** [comment]
I think the point I was trying to make is that it should be enabled by default for Linux so that the user doesn't have to switch it on manually. It's fine if you don't want to do that. But I like things more when the out of the box defaults just work.
